en/tutorials/index.md

```diff
@@ -38,11 +38,11 @@ Go ahead, add a couple of todos!
 
 #### 1.2 Syncing
 
-Once you're signed in to a Hoodie app, **Hoodie will constantly try to keep your user data in sync** between the server and any client that user may be signed in to. To see this happening, open the app's URL in a different Browser, or in incognito mode (so you won't be signed in automatically). Sign in to the second instance of the app with the same credentials, and when you add a todo in either tab, it should appear in the other within a few seconds. Keep that second tab open for now.
+Once you're signed in to a Hoodie app, **Hoodie will constantly try to keep your user data in sync** between the server and any client that user may be signed in to. To see this happening, open the app's URL in a different browser, or in incognito or private mode (so you won't be signed in automatically). Sign in to the second instance of the app with the same credentials, and when you add a todo in either tab, it should appear in the other within a few seconds. Keep that second tab open for now.
 
 #### 1.3 Offline support
 
-[We strongly believe that apps shouldn't break just because they're offline](/en/hoodieverse/hoodie-concepts.html#offline-first). So naturally, this todo app also works offline. Give it a try:
+[We strongly believe that apps shouldn't break just because they're offline](/en/hoodieverse/hoodie-concepts.md#offline-first). So naturally, this todo app also works offline. Give it a try:
```

Comment:
Same fix needed as the "first part of this guide" link. https://github.com/hoodiehq/documentation/pull/206/files#r58724226